assets/grafana/node-dashboard.json

```diff
@@ -405,15 +414,15 @@
           },
           "targets": [
             {
-              "expr": "(node_memory_MemFree{instance=~\"$server\"} / node_memory_MemTotal{instance=~\"$server\"}) * 100",
```

Comment:
`$server` is indeed a grafana interpolated thing. Now that you mention it, I agree, it should all work with `=` just fine. I'll try.